ls1intum/Artemis
# `General`: Keep assessment editor in read only mode when answering a feedback request

### Checklist
#### General
#### Client

### Motivation and Context
When answering a more feedback requests, it's not possible to modify the current assessment or add additional feedback items. They are not saved on the server. However, for feedback requests on automatically assessed programming exercises, the editor was still modifiable. This led to tutors adding feedback items and then referring to them in the answer text, causing students to not understand the incomplete answers. Some even tried to change the points during the feedback request.

### Description
On the student side, the explanation text telling that point changes are not possible is now no longer hidden behind a tooltip. This should make it more clear that changing the points is not possible.

When answering a feedback request, it's now not possible to add or edit feedback.

### Steps for Testing
Prerequisites:
- 1 Tutor
- 1 Student
- 1 Programming Exercise with automatic Complaints enabled

1. Submit a Feedback Request as a Student
2. Answer the request as a tutor. Verify that editing feedback is no longer possible.

### Review Progress

#### Code Review
#### Manual Tests

### Test Coverage
unchanged. Added a test that catches this case (fails when used against develop).

### Screenshots
old:
![grafik](https://user-images.githubusercontent.com/26540346/221422740-36cc9b22-d3b0-4b9e-874a-49a87d28b872.png)


new:
![grafik](https://user-images.githubusercontent.com/26540346/221422296-583bbd15-27fd-4283-aa5e-611bfe26590b.png)
